rebasehelper/helpers/koji_helper.py

```diff
@@ -288,13 +288,14 @@ def get_latest_build(cls, session, package):
         return None, None
 
     @classmethod
-    def download_build(cls, session, build_id, destination):
+    def download_build(cls, session, build_id, destination, pkg_type='RPM'):
```

Comment:
Why not make this method take list of architectures as an argument? And special case `src` as it has no logs?

Reply:
so `new-build/` directory content of `rebase-helper-results` would be changed from `RPM, SRPM` to `x86_64, src, aarch64..`

Reply:
No, nothing would change, only this method would accept list of architectures - `['noarch', 'x86_64']` in case of RPM build and `['src']` in case of SRPM build.